src/ray/core_worker/transport/task_receiver.h

```diff
@@ -117,50 +113,56 @@ class TaskReceiver {
   /// This should be called once for the actor creation task.
   void SetupActor(bool is_asyncio, int fiber_max_concurrency, bool execute_out_of_order);
 
- protected:
-  /// Cache the concurrency groups of actors.
-  // TODO(ryw): remove the ActorID key since we only ever handle 1 actor.
-  absl::flat_hash_map<ActorID, std::vector<ConcurrencyGroup>> concurrency_groups_cache_;
-
- private:
   /// The callback function to process a task.
   TaskHandler task_handler_;
+
   /// The event loop for running tasks on.
   instrumented_io_context &task_execution_service_;
+
   worker::TaskEventBuffer &task_event_buffer_;
+
+  /// Shared waiter for dependencies required by incoming tasks.
+  DependencyWaiter &waiter_;
+
   /// The language-specific callback function that initializes threads.
   std::function<std::function<void()>()> initialize_thread_callback_;
+
   /// The callback function to be invoked when finishing a task.
   OnActorCreationTaskDone actor_creation_task_done_;
-  /// Shared pool for producing new core worker clients.
-  std::shared_ptr<rpc::CoreWorkerClientPool> client_pool_;
-  /// Address of our RPC server.
-  rpc::Address rpc_address_;
-  /// Shared waiter for dependencies required by incoming tasks.
-  DependencyWaiter *waiter_ = nullptr;
+
   /// Queue of pending requests per actor handle.
   /// TODO(ekl) GC these queues once the handle is no longer active.
   absl::flat_hash_map<WorkerID, std::unique_ptr<SchedulingQueue>>
       actor_scheduling_queues_;
+
   // Queue of pending normal (non-actor) tasks.
   std::unique_ptr<SchedulingQueue> normal_scheduling_queue_ =
       std::make_unique<NormalSchedulingQueue>();
+
   /// The max number of concurrent calls to allow for fiber mode.
   /// 0 indicates that the value is not set yet.
   int fiber_max_concurrency_ = 0;
+
   /// If concurrent calls are allowed, holds the pools for executing these tasks.
   std::shared_ptr<ConcurrencyGroupManager<BoundedExecutor>> pool_manager_;
+
   /// If async calls are allowed, holds the fibers for executing async tasks.
   /// Only populated if this actor is async.
   std::shared_ptr<ConcurrencyGroupManager<FiberState>> fiber_state_manager_;
+
   /// Whether this actor use asyncio for concurrency.
   bool is_asyncio_ = false;
+
   /// Whether this actor executes tasks out of order with respect to client submission
   /// order.
   bool execute_out_of_order_ = false;
+
   /// The repr name of the actor instance for an anonymous actor.
   /// This is only available after the actor creation task.
   std::string actor_repr_name_;
+
+  /// Cache the concurrency groups of this worker's actor.
+  std::vector<ConcurrencyGroup> concurrency_groups_cache_;
```

Comment:
"cache" is a bit of a weird name because it is never modified/invalidated. maybe just call it `concurrency_groups_` and specify that it's only ever set once (at the time it has the creation task spec)?